### PR TITLE
BLOCKCHAIN-421 FE purge eth bridge code

### PR DESCRIPTION
**Pull request by MichalLustykNeti**

### CODE REVIEW

File: src/api/nodeRpcCall.js [hunk 1]
```diff
@@ -233,59 +233,11 @@ const getAdditionalAssets = async (address, isIndexNeed = false) => {
...
-  return bridge;
-};
-
 const provideJudgementAndAssets = async ({
   address, hash, walletAddress, merits, dollars,
 }) => {
-  merits = parseMerits(merits)
-  dollars = parseDollars(dollars)
+  const parsedMerits = parseMerits(merits);
+  const parsedDollars = parseDollars(dollars);
```
Comment (DorianSternVukotic) [inline src/api/nodeRpcCall.js:240]: Is this on purpose or some merge issue ?

File: src/components/Offices/Identity/index.js [hunk 2]
```diff
@@ -122,6 +125,9 @@ function parseEligibleOn(eligible_on) {
 }
 
 function EligibleOnAnalysis({ identity }) {
+  const blockNumber = useSelector(blockchainSelectors.blockNumber);
+  const [now] = useState(new Date());
+
```
Comment (DorianSternVukotic) [inline src/components/Offices/Identity/index.js:130]: Intentional ?

File: src/utils/walletHelpers.js [hunk 3]
```diff
@@ -84,3 +85,18 @@ export const formatAssetTransaction = (dollars_raw, asset, decimals, config = co
   decimals,
   config,
 );
+
+// FIXME this is generic, we should move it to walletHelpers and use everywhere
```
Comment (kacperzuk-neti) [inline src/utils/walletHelpers.js:89]: Drop the comment now that it's moved :D

**Review by kacperzuk-neti — CHANGES_REQUESTED** [review]

File: .env.dist [hunk 4]
```diff
@@ -16,9 +16,6 @@ REACT_APP_SSO_API_ADMIN_LINK='http://localhost:8080/signin?admin=true'
 # Dev Anvil
 REACT_APP_ETHER_CHAIN_ID=31337
 REACT_APP_ETHER_PROVIDER=http://127.0.0.1:8545
```
[the remark this answers is not in the document]
Comment (kacperzuk-neti) [reply .env.dist:18]: I'd go as far as removing it from package.json TBH.

File: src/api/nodeRpcCall.js (hunk 1 above)
Comment (MichalLustykNeti) [reply src/api/nodeRpcCall.js:240]: Yep does are eslint fixes

File: src/components/Offices/Identity/index.js (hunk 2 above)
Comment (MichalLustykNeti) [reply src/components/Offices/Identity/index.js:130]: Fixing eslint errors

File: src/api/nodeRpcCall.js (hunk 1 above)
Comment (DorianSternVukotic) [reply src/api/nodeRpcCall.js:240]: No they are not, linter will not rename merits to parsedMerits

**Review by kacperzuk-neti — APPROVED** [review]

**Pull request merged**